mne/time_frequency/psd.py

```diff
@@ -168,7 +168,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
     else:
         time_mask = Ellipsis
 
-    data = epochs.get_data()[:, picks][..., time_mask]
+    data = epochs.get_data()[:, picks][:, :, time_mask]
```

Comment:
This change has already been applied to the development version:

https://github.com/mne-tools/mne-python/blob/master/mne/time_frequency/psd.py#L140

But really what should (also) be done is to change `time_mask = Ellipsis` to `time_mask = slice(None)` above. It's better anyway.